docs/sources/mimir/references/http-api/index.md

```diff
@@ -976,10 +976,12 @@ Requires [authentication](#authentication).
 POST /api/v1/alerts
 ```
 
-Stores or updates the Alertmanager configuration for the authenticated tenant. The Alertmanager configuration is stored in the configured backend object storage.
+Stores or updates the Alertmanager configuration for the authenticated tenant. The Alertmanager configuration is stored in the configured backend object storage and shared between Alertmanager instances.
```

Comment:
> and shared between Alertmanager instances.

That seems like unnecessary addition. I mean, how else can it work in Mimir's horizontally scalable Alertmanager?

Reply:
that was probably just me but that sentence on its own felt like it's just leaking implementation details without a clear purpose (i.e. so what if the config ends up in object storage). But maybe I was too tired when writing this. I can revert this 
```suggestion
Stores or updates the Alertmanager configuration for the authenticated tenant. The Alertmanager configuration is stored in the configured backend object storage.
```